hera_cal/delay_filter.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+from hera_cal import io
+from pyuvdata import UVData
+from hera_cal.datacontainer import DataContainer
+from collections import OrderedDict as odict
+from uvtools.dspec import delay_filter
+from copy import deepcopy
+from scipy import constants
+
+class Delay_Filter():
+
+    def __init__(self):
+        '''Class for loading data, performing uvtools.dspec.delay_filter, and writing out data using pyuvdata.
+        To use, run either self.load_data() or self.load_dicts(), then self.run_filter(). If data is loaded with a single 
+        string path or a single UVData object, it can be written to a new file using self.write_filtered_data().
+        '''
+        self.writable = False
+
+
+    def load_data(self, input_data, format='miriad'):
+        '''Loads in and stores data for delay filtering. 
+
+        Arguments:
+            input_data: data file path, or UVData instance, or list of either strings of data file paths 
+                or list of UVData instances to concatenate into a single internal DataContainer
+            format: file format of data. Default 'miriad.' Ignored if input_data is UVData object(s).
+        '''
+        if isinstance(input_data, (str,UVData)):
+            self.writable = True
+            self.input_data, self.format = input_data, format
+        self.data, self.flags, self.antpos, _, self.freqs, self.times, _, _ = io.load_vis(input_data, return_meta=True, format=format)
+
+
+    def load_dicts(self, data, flags, freqs, antpos):
```

Comment:
Was there a reason you split the load_data and load_dicts function? Can we combine them?

Reply:
rename function to load_data_dicts. Makes it a bit clear.

Reply:
I'm renaming load_dicts to load_data_as_dicts to be more clear.